app/src/main/java/net/squanchy/speaker/view/SpeakersView.java

```diff
@@ -0,0 +1,44 @@
+package net.squanchy.speaker.view;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.AttributeSet;
+
+import java.util.List;
+
+import net.squanchy.speaker.Speaker;
+
+public class SpeakersView extends RecyclerView {
+
+    private SpeakerAdapter adapter;
+
+    public SpeakersView(Context context) {
+        super(context);
+    }
+
+    public SpeakersView(Context context, @Nullable AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+
+        LinearLayoutManager layoutManager = new LinearLayoutManager(getContext());
+        setLayoutManager(layoutManager);
+        adapter = new SpeakerAdapter();
+        setAdapter(adapter);
+        setClipToPadding(false);
```

Comment:
This can/should be set using the default style (see `int defStyleAttr` in the missing ctor)